salt/modules/cmdmod.py

```diff
@@ -1156,6 +1192,20 @@ def run(
 
       .. versionadded:: 2019.2.0
 
+    :param list success_stdout: This parameter will be allow a list of
```

Comment:
```suggestion
    :param list success_stdout: This parameter will allow a list of
```

Reply:
This edit and the versionadded change suggestion apply to all of the rest of the PR.

Reply:
Done